srcpkgs/chromium/template

```diff
@@ -121,37 +117,21 @@ do_configure() {
 
 	python2 third_party/libaddressinput/chromium/tools/update-strings.py
 
-	# bootstrap NaCL toolchain
-	if [ -n "$build_option_nacl" ] && [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
-		# XXX xtraeme: fix gcc{,-c++}-multilib instead.
-		ldconfig &>/dev/null
-		# Download NaCL toolchains
-		python2 nbuild/download_acl_toolchains.py \
-			--packages nacl_x86_newlib,pnacl_newlib,pnacl_translator sync --extract
-		# donwload clang
-		python2 tools/clang/scripts/update.py
-		ln -s /usr/lib/libncursesw.so.6 third_party/llvm-build/Release+Asserts/lib/libtinfo.so.5
-	fi
-
-	# bootstrap GN
 	bootstrap_conf="is_clang=$(vopt_if clang true false)"
-	bootstrap_conf+='
-		treat_warnings_as_errors=false
-		binutils_path="/usr/bin"
-		gold_path="/usr/bin/ld.gold"
-		use_sysroot=false
-		use_allocator="none"
-		use_allocator_shim=false'
+
+	bootstrap_conf+=' treat_warnings_as_errors=false binutils_path="/usr/bin"'
+	bootstrap_conf+=' gold_path="/usr/bin/ld.gold" use_sysroot=false'
+	bootstrap_conf+=' use_allocator="none" use_allocator_shim=false'
 
 	# XXX: gold broken with musl
 	case "${XBPS_TARGET_MACHINE}" in
-	*-musl) bootstrap_conf+=' use_gold=false' ;;
-	*) bootstrap_conf+=' use_gold=true' ;;
+		*-musl) bootstrap_conf+=' use_gold=false' ;;
+		*) bootstrap_conf+=' use_gold=true' ;;
 	esac
 
 	AR="ar" CC=$CC_FOR_BUILD CXX=$CXX_FOR_BUILD LD=$CXX_FOR_BUILD \
-		python2 tools/gn/bootstrap/bootstrap.py -s -v --no-clean \
```

Comment:
is `--no-clean` gone?

Reply:
No, but I don't see the advantage of using it here - rebuilding GN everytime we call bootstrap.py isn't a bad thing for one-time-builds and is really nice for subsequent builds (e.g. when debugging why GN doesn't build)

Reply:
Takes less time because we delete everything at the end anyways.

Reply:
No clean only cleans when a _new_ GN build is built if I understand it correctly, so it doesn't have any effect on successful builds, it only affects subsequent builds (e.g. if the build failed and a packager attempts to run the build again - with different params).

Reply:
Ok looks like it was dropped in https://chromium.googlesource.com/chromium/src/+/fcbf5f6262210e3ed7eb30f0265aff9aa615e27b%5E%21/#F1 and makes no difference now.
They just kept it for compatibility, seems like `-s` is obsolete too, if this patch is in the release.

Reply:
I don't think that `-s` is obselete yet.

Reply:
https://chromium.googlesource.com/chromium/src/+/69.0.3497.112/tools/gn/bootstrap/bootstrap.py

Looks ignored too.